### PR TITLE
2766: Translation of tutorial to czech

### PR DESCRIPTION
**Pull request by simsa-st**
modified:   tutorial.txt

### CODE REVIEW

File: dev/_sources/tutorial.txt [hunk 1]
```diff
@@ -280,16 +276,16 @@ To combine things back together, use ``together(expr, x)``::
 
 .. index:: calculus
 
-Calculus
+Kalkulus
```
Comment (certik) [inline dev/_sources/tutorial.txt:279]: I think the Czech translation (at least when I was studying a few years ago) is either "diferenciální/integrální počet" or "analýza".

File: dev/_sources/tutorial.txt [hunk 2]
```diff
@@ -76,21 +76,20 @@ distribution, e.g.:
...
 --------------
 
-For experimenting with new features, or when figuring out how to do things, you
-can use our special wrapper around IPython called ``isympy`` (located in
-``bin/isympy`` if you are running from the source directory) which is just a
-standard python shell that has already imported the relevant sympy modules and
-defined the symbols x, y, z and some other things:
+Při zkoušení nových funkcí nebo když se snažíte něčemu přijít na kloub, můžete
+použít na IPythonu založenou ``isympy`` (umístěnou v ``bin/isympy``, pokud jí
+spouštítě ze složky se zdrojovým kódem) což je prostě standardní Python shell,
```
Comment (certik) [inline dev/_sources/tutorial.txt:90]: spouštítě -> spouštíte

File: dev/_sources/tutorial.txt [hunk 3]
```diff
@@ -76,21 +76,20 @@ distribution, e.g.:
...
+který má zapojoné související sympy moduly a definoval symoboly x, y, z
```
Comment (certik) [inline dev/_sources/tutorial.txt:91]: zapojené

File: dev/_sources/tutorial.txt [hunk 4]
```diff
@@ -116,17 +115,18 @@ defined the symbols x, y, z and some other things:
 
 .. note::
 
-    Commands entered by you are bold. Thus what we did in 3 lines in a regular
-    Python interpreter can be done in 1 line in isympy.
+    Vámi zadané příkazy jsou tučně. Vidíme, že to co bychom udělali ve
```
Comment (certik) [inline dev/_sources/tutorial.txt:118]: comma before "co".

File: dev/_sources/tutorial.txt [hunk 5]
```diff
@@ -116,17 +115,18 @@ defined the symbols x, y, z and some other things:
...
+    3 řádcích v běžném interpreteru Pythonu lze udělat v isympy v 1 řádku.
```
Comment (certik) [inline dev/_sources/tutorial.txt:119]: comma before "lze" (I think at least)

File: dev/_sources/tutorial.txt [hunk 6]
```diff
@@ -142,41 +142,37 @@ So Rational(1,2) represents 1/2, Rational(5,2) represents 5/2, and so on.
...
-We also have some special constants, like e and pi, that are treated as symbols
-(1+pi won't evaluate to something numeric, rather it will remain as 1+pi), and
-have arbitrary precision::
+Také máme k dispozici několik konstant, jak e nebo pí, se kterými se pracuje
+jako se symboly (1+pi se nevypočítá jako numerická hodnota, ale ponechá se 1+pi)
+a mají libovolnou přesnost::
 
     >>> pi**2
-    pi**2
+    pi**
```
Comment (certik) [inline dev/_sources/tutorial.txt:175]: You need to leave the original `pi**2` here.

File: dev/_sources/tutorial.txt [hunk 7]
```diff
@@ -184,28 +180,27 @@ have arbitrary precision::
...
     >>> oo + 1
     oo
 
-Symbols
+Symboly
 -------
 
-In contrast to other Computer Algebra Systems, in SymPy you have to declare
-symbolic variables explicitly::
+V kontrastu oproti jiným počítačově algebraickým systémům, v SymPy je třeba
```
Comment (certik) [inline dev/_sources/tutorial.txt:195]: I would write: "narozdíl od jiných počítačových algebraických systémů". Let me know if you like it better too.

File: dev/_sources/tutorial.txt [hunk 8]
```diff
@@ -227,18 +223,18 @@ They can be substituted with other numbers, symbols or expressions using ``subs(
...
+Pro celý zbytek tutoriálu předpokládejme, že jsme zadali::
 
     >>> init_printing(use_unicode=False, wrap_line=False, no_global=True)
 
-This will make things look better when printed. See the :ref:`printing-tutorial`
-section below. If you have a unicode font installed, you can pass
-use_unicode=True for a slightly nicer output.
+To nám zajistí hezčí výstup. Koukněte do sekce níže :ref:`printing-tutorial`.
+Pokud máte nainstalovaný font unicode, můžete nastavit ``use_unicode=True`` pro
+o něco hezčí výstup.
```
Comment (certik) [inline dev/_sources/tutorial.txt:232]: Here I would maybe write "nastavením `use_unicode=True` dostanete o něco hezčí výstup." Let me know what you think.

File: dev/_sources/tutorial.txt [hunk 9]
```diff
@@ -280,16 +276,16 @@ To combine things back together, use ``together(expr, x)``::
...
 .. index:: limits
 
-Limits
+Limity
 ------
 
-Limits are easy to use in sympy, they follow the syntax ``limit(function,
-variable, point)``, so to compute the limit of f(x) as x -> 0, you would issue
+Limity se používají v sympy jednoduše, pomocí syntaxe ``limit(function,
+variable, point)``, takže pro vypočtení limity f(x) pro x -> 0, byste dosadili
```
Comment (certik) [inline dev/_sources/tutorial.txt:288]: Delete the comma before "byste" ?

File: dev/_sources/tutorial.txt [hunk 10]
```diff
@@ -646,15 +642,15 @@ In ``isympy``::
 
 .. index:: linear algebra
 
-Linear Algebra
-==============
+Lineární Algebra
```
Comment (certik) [inline dev/_sources/tutorial.txt:645]: algebra

File: dev/_sources/tutorial.txt [hunk 11]
```diff
@@ -698,33 +693,33 @@ substitutions, as follows::
...
-If the match is unsuccessful, it returns ``None``::
+Pokud je porovnání neúspěšné, vrátí ``None``::
 
     >>> print (x+1).match(p**x)
     None
 
-One can also use the exclude parameter of the ``Wild`` class to ensure that
-certain things do not show up in the result::
+Lze také použít parametr ``exclude`` třídy ``Wild``, aby se určité věci
+nezobrazili ve výsledku::
```
Comment (certik) [inline dev/_sources/tutorial.txt:702]: nezobrazily

File: dev/_sources/tutorial.txt [hunk 12]
```diff
@@ -758,8 +753,8 @@ This is a nice ascii-art printing produced by a ``pprint`` function:
      |
     /
 
-If you have a unicode font installed, it should use unicode pretty printing by default.
-You can override this using the ``use_unicode`` option.:
+Pokud máte nainstalované písmo unicode, měla by ho funkce automaticky využívat.
+Vyoužívání unicode písma se dá nastavit změnou ``use_unicode``::
```
Comment (certik) [inline dev/_sources/tutorial.txt:757]: Využívání

File: dev/_sources/tutorial.txt [hunk 13]
```diff
@@ -768,11 +763,11 @@ You can override this using the ``use_unicode`` option.:
...
 
 
-See also the wiki `Pretty Printing
-<https://github.com/sympy/sympy/wiki/Pretty-Printing>`_ for more examples of a nice
-unicode printing.
+Pro více příkladů pěkného unicode výstupu koukněte na wiki na `Pretty Printing
+<https://github.com/sympy/sympy/wiki/Pretty-Printing>`_
 
-Tip: To make the pretty printing default in the python interpreter, use::
+Tip: Pokud chcete v interpreteru Pythonu pěkné vypisování používat defaultně,
```
Comment (certik) [inline dev/_sources/tutorial.txt:769]: interpretru?

File: dev/_sources/tutorial.txt [hunk 14]
```diff
@@ -841,40 +834,46 @@ Tip: To make the pretty printing default in the python interpreter, use::
...
 .. image:: pics/pngview1.png
 
-Notes
------
+Poznámky
+--------
 
-``isympy`` calls ``pprint`` automatically, so that's why you see pretty
-printing by default.
+``isympy`` volá ``pprint`` automaticky, proto vidíte pěkný výpis defaultně.
```
Comment (certik) [inline dev/_sources/tutorial.txt:850]: "proto standardně vidíte pěkný výpis". Maybe?

File: dev/_sources/tutorial.txt [hunk 15]
```diff
@@ -76,21 +76,20 @@ distribution, e.g.:
...
-For other means how to install SymPy, consult the  Downloads_ tab on the
-SymPy's webpage.
+Pro další způsoby instalace SymPy si prohlédněte záložku Downloads_ na stránkách
+SymPy.
 
 .. _Downloads: http://code.google.com/p/sympy/wiki/DownloadInstallation?tm=2
 
 
-isympy Console
+isympy Konzole
```
Comment (certik) [inline dev/_sources/tutorial.txt:85]: konzole

**Comment by certik** [comment]
Thanks. After you fix all these issues, please rename the file to tutorial.cs.txt and send the pull request against the sympy/ repository. You'll find the original tutorial.txt in doc/src/tutorial.txt

**Comment by certik** [comment]
(Feel free to discuss any of my corrections above if you disagree.)

File: dev/_sources/tutorial.txt (hunk 7 above)
Comment (simsa-st) [inline dev/_sources/tutorial.txt:195]: Yes, I think, you are right, it's better.

File: dev/_sources/tutorial.txt (hunk 8 above)
Comment (simsa-st) [inline dev/_sources/tutorial.txt:232]: Yes, my sentence sounds little weird. Maybe, that: "volbou `use_unicode=True` dostanete o něco hezčí výstup" would be even better. Please tell me, what is your opinion.

File: dev/_sources/tutorial.txt (hunk 13 above)
Comment (simsa-st) [inline dev/_sources/tutorial.txt:769]: Oh, you're right: http://macek.sandbox.cz/texty/python-tutorial-cz/tut/node4.html
I wrote it on other places too, so I will change it there as well.

**Comment by simsa-st** [comment]
Okay, I made, what you had told me. I hope, that correctly. Please let me now, if I should do something more.

File: dev/_sources/tutorial.txt (hunk 8 above)
Comment (certik) [inline dev/_sources/tutorial.txt:232]: Yes, definitely, this is even better. Thanks!

**Comment by certik** [comment]
I see your new pull request here, so I'll review it over there.

https://github.com/sympy/sympy/pull/796

I am closing this one.

**Pull request closed without merging**